pkg/render/manager.go

```diff
@@ -80,6 +82,7 @@ func Manager(
 			ManagerSecretCertName,
 			825*24*time.Hour, // 825days*24hours: Create cert with a max expiration that macOS 10.15 will accept
 			nil,
+			ManagerServiceExternalDNS, ManagerServiceInternalDNS,
```

Comment:
On closer inspection, I guess this was intentional, since the function is variadic with the variable amount of hostnames `hostnames ...string`. 

But in that case you could use a slice and then use "..." to make it clearer. 

```suggestion
	hostnames := []string{ManagerServiceExternalDNS, ManagerServiceInternalDNS}
	...
	tlsKeyPair, err = CreateOperatorTLSSecret(nil,
			ManagerTLSSecretName,
			ManagerSecretKeyName,
			ManagerSecretCertName,
			825*24*time.Hour, // 825days*24hours: Create cert with a max expiration that macOS 10.15 will accept
			nil,
			hostnames...,
		)
```

Reply:
Hm ... I take the thumbs up back. :) I do not think we need to create a slice. The purpose of a variadic function is to allow the caller to use any numbers of parameters.

Reply:
Yes, the variadic part of this is the hostnames, which can be any number of them. 

I was saying that if you intended to have both ManagerServiceExternalDNS and ManagerServiceInternalDNS on the same line to indicate they are both hostnames, then a slice with the `...` allows you to communicate that more explicitly. 

You can ignore what I said if that wasn't the intention. Either way it's minor.